cooltools/api/snipping.py

```diff
@@ -203,7 +198,7 @@ def _pileup(data_select, data_snip, arg):
     return np.dstack(stack), feature_group["_rank"].values
 
 
-def pileup_legacy(features, data_select, data_snip, map=map):
+def select_snip(features, data_select, data_snip, map=map):
```

Comment:
looks like an internal-enough function to become `_select_snip` at the very least

Reply:
can we call it `_pileup` - why not ?

Reply:
hide existing `_pileup` inside of this function, and rename that internal func to `_extract_pileup`/`_create_stack`/`extract_Stack` ...

and then reuse the name `_pileup` for *this* function (currently `select_snip`)
@agalitsyna what do you think ?

Reply:
I don't want to put a function inside a function tbh, but renaming is great, thanks.

Reply:
renaming is great, I keep using this function as if it's not internal